compose.yml

```diff
@@ -13,6 +13,8 @@ services:
         condition: service_healthy
```

Comment:
nitpick(unrelated to this PR):  using the `service_healthy` is a really good idea! But healthchecks are required on the "target" dependency, otherwise Docker considers that `service_healthy === service_started`.

In this particular case, the `db` service does not have an healthcheck defined: https://github.com/docker-library/postgres/blob/56a1986772dd0f9488d54dccb82427c0db0b0599/14/alpine/Dockerfile
`
But you can add one to the `db` service through docker-compose to make this really strong and keep the initial intent (e.g. only starting the Java app once you're assurred that the DB service is up) with this directive: https://docs.docker.com/compose/compose-file/compose-file-v3/#healthcheck .

The rationale behind "NOT" defining a default healthcheck for official image is because the "healthcheck" is a Docker-only feature: it will be ignored on Kubernetes, containerd, podman, etc. 
Kubernetes, for instance, has the concept of "probes" instead, which are fundamentally different: Kubernetes porbes are executed by a process *outside* the container (`kubelet`), while the Docker healthcheck are executed *inside* the container (hence the need to make sure that `curl`, or `wget` or any tools is installed in your image).

Reply:
🤔 I'm thinking a bit in advance but this comment might not be so much nitpick:

- The target is Kubernetes for the application. But we'll most probably use a postgres managed database outside Kubernetes (e.g. not run in a container, neither in the same orchestrator).

=> so maybe you want to remove the `depends_on` directive here at all (*when* you'll be ready, of course :)) to simulate what could happen in production: the application container being up while the postgres database is not started or reachable for a few seconds.
We expect the application to crash (most probably), so a ['restart' policy](https://docs.docker.com/compose/compose-file/compose-file-v3/#restart) could be required.

Reply:
https://github.com/jenkins-infra/plugin-health-scoring/pull/36/files#diff-3493e6b5ddf34891e572f911db893efd9e46af828e011ea778a7c1eb64763588R24-R28

Reply:
as pointed by @alecharp , there is already an health check on the db: I forgot to unfold 🤦 

Sorry for the noise